src/foam/u2/ModalHeader.js

```diff
@@ -31,12 +31,14 @@ foam.CLASS({
       border-bottom: solid 1px #CBCFD4;
     }
     ^title{
-      font-family: /*%FONT1%*/ Roboto, 'Helvetica Neue', Helvetica, Arial, sans-serif;
-      font-size: 18px;
+      font-family: IBM Plex Sans;
```

Comment:
Why are we not using the font from theme? 
```suggestion
      font-family: /*%FONT1%*/ 'IBM Plex Sans'; 
```